repose-aggregator/core/repose-core/src/main/java/org/openrepose/core/services/reporting/metrics/MetricsJmxObjectNameFactory.java

```diff
@@ -23,12 +23,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.inject.Named;
 import javax.management.MalformedObjectNameException;
 import javax.management.ObjectName;
 import java.util.Hashtable;
 
-@Named
```

Comment:
Let's go ahead with the annotated bean, we are far enough ahead in this sprint that doing this right is worth the time. I'd make sure anything that needs it is looking for the interface and not this specific class, then in the couple of tests that look in a different package you can just stub out an implementation.